### PR TITLE
 Revert "added Address and User to models.py and"

### PR DESCRIPTION
**Pull request by jagz97**
Reverts jagz97/Cmpe131-Team-6#8

### CODE REVIEW

**Comment by jagz97** [comment]
@HectorSal There were issues with models. Need to provide foreign key for database relationships. Throwing these errors:

Screen Shot 2022-04-30 at 8 46 55 PM

File: app/forms.py [hunk 1]
```diff
@@ -13,18 +13,4 @@ class Products(Form):
 
     image = FileField('Image 1', validators=[FileRequired(), FileAllowed(['png','gif','jpg','jpeg'])])
     image1 = FileField('Image 2', validators=[FileRequired(), FileAllowed(['png','gif','jpg','jpeg'])])
-    image2 = FileField('Image 3', validators=[FileRequired(), FileAllowed(['png','gif','jpg','jpeg'])])
-class LoginForm(Form):
-    username = StringField('Username', validators=[DataRequired()])
-    password = PasswordField('Password', validators=[DataRequired()])
```
Comment (jagz97) [inline app/forms.py:19]: Missing imports Password Flied and Boolean Field

Comment (jagz97) [reply app/forms.py:19]: @HectorSal

File: app/models.py [hunk 2]
```diff
@@ -8,22 +8,4 @@ class Category(db.Model):
...
-    state_province_region = db.Column(db.String(128), nullable=False)
-    zip_postal_code = db.Column(db.String(32), nullable=False)
-    country = db.Column(db.String(64), nullable=False)
-    user = db.relationship('User', back_populates='Address')
-
-class User(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
-    username = db.Column(db.String(30), index=True, nullable=False, unique=True)
-    email = db.Column(db.String(30), index =True, nullable=False, unique=True)
-    password_hash = db.Column(db.String(128)
```
Comment (jagz97) [inline app/models.py:26]: syntax error

Comment (jagz97) [reply app/models.py:26]: @HectorSal

File: app/models.py [hunk 3]
```diff
@@ -8,22 +8,4 @@ class Category(db.Model):
...
-    shipping_address = db.relationship('Address', back_populates='User', uselist=False)
```
Comment (jagz97) [inline app/models.py:27]: need to provide Foreign key for db relationship
@HectorSal

**Review by HectorSal — APPROVED** [review]

**Pull request merged**